src/components/carbon-provider/carbon-provider.spec.js

```diff
@@ -23,4 +25,27 @@ describe("CarbonProvider", () => {
 
     expect(wrapper.find(ThemeProvider).prop("theme")).toBe(aegeanTheme);
   });
+
+  it("persists the parent validationRedesignOptIn value", () => {
```

Comment:
There shouldn't be a parent value, the CarbonProvider should be at the root of the app or MFE - unless I'm missing something?

Reply:
Yeah it's overkill I'll get it removed thanks